CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Please drop the CoC, everyone is adults and should behave as such, these are useless.